apisix/admin/init.lua

```diff
@@ -14,6 +14,7 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
+local filter = require("apisix.router").filter
```

Comment:
Does module `apisix.router` export the function `filter`?
BTW, the `require` call should be put under `local require = require`.